submissions/drewzag/task_js_dom/index.js

```diff
@@ -0,0 +1,47 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const getData = async () =>
+    fetch('/src/data.json')
+      .then((res) => {
+        if (res.ok) return res.json()
+        else throw new Error('Something went wrong')
+      })
+      .catch((error) => {
+        console.log(error)
+      })
+
+  const list = document.querySelector('.list')
+  const main = document.querySelector('.main')
+  const h1 = document.querySelector('h1')
+  const menuIcon = document.querySelector('div.menu__icon')
+
+  getData().then((res) => {
+    res.forEach((el) => {
+      const li = document.createElement('li')
+      li.classList.add('list__item')
+      li.textContent = el.team
+      list.appendChild(li)
+      li.addEventListener('click', onClickHandlerItem)
```

Comment:
Use [event delegation](https://davidwalsh.name/event-delegate) instead of adding multiple event listeners